src/vivarium_nih_us_cvd/plugins/causes_parser.py

```diff
@@ -54,17 +71,33 @@ def get_components(
             A list of initialized components.
         """
         components = []
-        if isinstance(component_config, ConfigTree) and "causes" in component_config:
+
+        if not component_config:
+            return components
+
+        if "external_configuration" in component_config:
```

Comment:
How is it guaranteed that the external_configurations will be parsed prior to the causes loop below?

Reply:
This is guaranteed, because we check for it first here. There isn't any parsing that happens prior to this line.